privacyidea/lib/tokens/u2f.py

```diff
@@ -110,12 +110,12 @@ def parse_registration_data(reg_data, verify_cert=True):
     :return: tuple
     """
     reg_data_bin = url_decode(reg_data)
-    reserved_byte = six.int2byte(six.indexbytes(reg_data_bin, 0))  # must be '\x05'
-    if reserved_byte != b'\x05':
-        raise Exception("The registration data is in a wrong format. It must"
+    reserved_byte = reg_data_bin[0]  # must be '\x05'
```

Comment:
since we do not return the '\x05' as a byte but as an int '5', should we rename the variable?
Or should we, for conformity to line 87 use "bytes()", even if it is not necessary?
I only realized this, since you explained this 5 minutes ago.

Reply:
Done in [436cb0a](https://github.com/privacyidea/privacyidea/pull/3554/commits/436cb0a226ee69e7467460fd04713088e51649fc)